src/app/shares/viewshareaccount/viewshareaccount.component.html

```diff
@@ -0,0 +1,162 @@
+<div class="container">
+  <span class="heading">
+    <strong>{{ shareAccountDetails.productName }}(#{{ shareAccountDetails.accountNo }})</strong>
+  </span>
+  <hr/>
+  <div class="pull-right btn-group">
+    <button *ngFor="let button of buttons.singlebuttons"
+      (click)="clickEvent(button.name, shareAccountDetails.id)" mat-raised-button color="primary" class="btn-group" ><i
+        class="{{ button.icon }}"></i> {{ button.name }}</button>
+    <button mat-raised-button color="primary" [matMenuTriggerFor]="more">More <i class="fa fa-angle-down"></i></button>
+    <mat-menu #more="matMenu">
+      <button *ngFor="let button of buttons.options" (click)="clickEvent(button.name, shareAccountDetails.id)" mat-menu-item>{{ button.name }}</button>
+    </mat-menu>
+  </div>
+  <div class="col-sm-12 col-md-12">
+    <div class="row">
+      <div class="col-sm-6 col-md-6">
+        <table mat-table [dataSource]="accountTableDataSource" class="table table-bordered table-striped">
```

Comment:
Update this using mat-card instead. To maintain consistency, this should be similar to when we view an accounting rule. Two cards side by side in case of larger screens and stacked in smaller screens. Remove any unnecessary divs and classes after that.